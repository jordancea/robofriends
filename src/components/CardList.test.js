import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import CardList from "./CardList";

it("should render the CardList component", () => {
  const mockRobots = [{ id: 1, name: "Test Name 2", email: "test@test.com" }];
  expect(toJson(shallow(<CardList robots={mockRobots} />))).toMatchSnapshot();
});
