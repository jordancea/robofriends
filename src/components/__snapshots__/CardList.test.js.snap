// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the CardList component 1`] = `
<div>
  <Card
    email="test@test.com"
    id={1}
    key="0"
    name="Test Name 2"
  />
</div>
`;
